src/actions.ts

```diff
@@ -2,9 +2,10 @@ import { When } from '@cucumber/cucumber';
 import { Element, ElementArray } from 'webdriverio';
 import { conditionValidations, conditionWait } from './conditionWait';
 import { getValue, getElement } from './transformers';
-import {parseCoords, Keys, parseKeySequence, parseCoordsAsObject} from './utils';
+import {parseCoords, parseKeySequence, parseCoordsAsObject} from './utils';
 import { click, doubleClick, rightClick } from './click';
 import { dragAndDrop } from './utils';
+import memory from "@qavajs/memory";
```

Comment:
do not use memory directly, there is getValue function that already imported to file